<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KDEPlotDock</class>
 <widget class="QWidget" name="KDEPlotDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="11" column="0">
        <spacer name="verticalSpacerGeneral">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>214</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lKernelType">
         <property name="text">
          <string>Kernel:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item row="9" column="2">
        <widget class="QComboBox" name="cbPlotRanges"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lBandwidth">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lBandwidthType">
         <property name="text">
          <string>Bandwidth:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment"/>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="cbKernelType"/>
       </item>
       <item row="6" column="2">
        <widget class="NumberSpinBox" name="sbBandwidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lDataColumn">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lPlotRange">
         <property name="text">
          <string>Plot range:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="cbBandwidthType"/>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="lFuncPic">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" rowspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="chkLegendVisible">
         <property name="text">
          <string>Legend</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabKDE">
      <attribute name="title">
       <string>Estimation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="lEstimationLine">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Line</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lEstimationFilling">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Filling</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>654</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMargins">
      <attribute name="title">
       <string>Margins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lRug">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rug Plot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lRugEnabled">
         <property name="text">
          <string>Enabled:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="chkRugEnabled">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lRugLength">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="NumberSpinBox" name="sbRugLength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lRugWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="NumberSpinBox" name="sbRugWidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lRugOffset">
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="NumberSpinBox" name="sbRugOffset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>543</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
